### PR TITLE
Add YAF extension

### PR DESCRIPTION
**Pull request by baikit**
## Description

I added an extension to my branch, which is YAF

## Motivation and Context

I think YAF is a good PHP extension framework. Its performance is very good. And many developers are using this framework.

## Types of Changes
The way I submit to add YAF this time is to install it directly from PECL. And different version can be installed according to the configuration

## Definition of Done Checklist:

- [] I've read the [Contribution Guide](http://laradock.io/contributing).
- [] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

**Comment by baikit** [comment]
i test by actions  and all success

File: php-fpm/php7.2.ini [hunk 1]
```diff
@@ -936,7 +936,7 @@ cli_server.color = On
 [Date]
 ; Defines the default timezone used by the date functions
 ; http://php.net/date.timezone
-;date.timezone =
+date.timezone = "Asia/Shanghai"
```
Comment (bestlong) [inline php-fpm/php7.2.ini:939]: not everyone uses this timezone.

File: env-example [hunk 2]
```diff
@@ -69,7 +73,9 @@ COMPOSE_CONVERT_WINDOWS_PATHS=1
 ### Environment ###########################################
 
 # If you need to change the sources (i.e. to China), set CHANGE_SOURCE to true
-CHANGE_SOURCE=false
+
+CHANGE_SOURCE=true
+
```
Comment (bestlong) [inline env-example:78]: This adjustment should be to modify .env

Comment (baikit) [reply env-example:78]: got it

File: php-fpm/php7.2.ini (hunk 1 above)
Comment (baikit) [reply php-fpm/php7.2.ini:939]: got it

**Comment by baikit** [comment]
How else do I need to modify my pull request to merge into the master.

**Pull request closed without merging**